java/KafkaConfigProviders/Kafka-mTLS-Keystore-Sql-ConfigProviders/README.md

```diff
@@ -0,0 +1,105 @@
+## Sample illustrating how to use MSK config providers in Flink Kafka connectors, for mTLS authentication
+
+* Flink version: 1.20
+* Flink API: Table API & SQL
+* Language: Java (11)
+* Flink connectors: Kafka (mTLS authentication)
+
+This sample illustrates how to configure the Flink Kafka connectors (KafkaSource and KafkaSink) 
```

Comment:
nit: "(source and sink)" 
KafkaSource and KafkaSink are the name of the connectors in DataStream.

Reply:
Updated text